CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ endif()
 find_package(dispatch QUIET)
 find_package(Foundation QUIET)
 find_package(SQLite3 REQUIRED)
+find_package(libzip REQUIRED)
```

Comment:
This is different from zlib, right? I was thinking since we already depend on zlib and have plans to add it to `update-checkout-config.json` to always build it with the toolchain, maybe we could rely on that one? That's assuming it provides all of the necessary functionality.